api/tests/routes/backoffice/finance_test.py

```diff
@@ -1058,8 +1058,17 @@ class GetOverpaymentIncidentTest(GetEndpointHelper):
         "venue_factory,expected_xpf_text",
         [(offerers_factories.VenueFactory, ""), (offerers_factories.CaledonianVenueFactory, "(1205 CFP)")],
     )
-    def test_get_incident(self, authenticated_client, venue_factory, expected_xpf_text):
-        finance_incident = finance_factories.FinanceIncidentFactory(venue=venue_factory())
+    @pytest.mark.parametrize(
+        "zendesk_id,expected_zendesk_id",
+        [
+            (None, None),
+            ("1", "1"),
+        ],
+    )
```

Comment:
Je pense que t'es pas obligé de garder expected_zendesk_id, et je mettrais `{zendesk_id}" in content` plus bas.
Pareil pour les autres tests